cmd/clusterctl/api/v1alpha3/metadata.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kubebuilder:object:root=true
+
+// Metadata for a provider repository
+type Metadata struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// +optional
+	ReleaseSeries []ReleaseSeries `json:"releaseSeries"`
+}
+
+// ReleaseSeries mapa a provider release series (major/minor) with a ClusterAPIVersion
+type ReleaseSeries struct {
+	// +optional
+	Major uint `json:"major,omitempty"`
+
+	// +optional
+	Minor uint `json:"minor,omitempty"`
```

Comment:
Shouldn't this be required? Otherwise, if there is a reason for it to be optional, it should be a pointer so that we can differentiate unset from 0.